stable/prometheus-operator/templates/prometheus/serviceperreplica.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if and .Values.prometheus.enabled .Values.prometheus.servicePerReplica.enabled }}
+{{- $count := .Values.prometheus.prometheusSpec.replicas | int -}}
+{{- $serviceValues := .Values.prometheus.servicePerReplica -}}
+{{ range $i, $e := until $count }}
+---
```

Comment:
This should be a list, rather than individual resources

Reply:
I've updated both the service and ingress resources to be `ServiceList` and `IngressList` kinds as I took that to be what you meant.

For my own interest, is there any particular reason to prefer these over a templated list of separate resources like what I had originally?